src/Apps/ViewingRoom/routes.tsx

```diff
@@ -39,6 +37,7 @@ export const routes: RouteConfig[] = [
       {
         path: "/works",
         Component: WorksRoute,
+        ignoreScrollBehavior: true,
```

Comment:
I'm going to spend time this week and get proper types into our routes. Right now things are free-form.